image/oci.go

```diff
@@ -12,12 +12,18 @@ import (
 	"github.com/pkg/errors"
 )
 
+type descriptorOCI1 struct {
```

Comment:
My guess is that, at the time, the spec was moving fast and breaking things?  I wouldn't object to switching to using them directly.